plugins/scaffolder/src/components/TemplateCard/TemplateCard.tsx

```diff
@@ -74,6 +74,12 @@ const useStyles = makeStyles(theme => ({
   leftButton: {
     marginRight: 'auto',
   },
+  starButton: {
```

Comment:
Just decided to keep the styles for now, the `TemplateCard` is going to be re-written anyways.